plugins/workspace-minimap/README.md

```diff
@@ -15,6 +15,25 @@ npm install @blockly/workspace-minimap --save
 ```
 
 ## Usage
```

Comment:
Can you add a picture above that shows the minimap in action?

I would also add some info to the top-level blurb that explains what the focus mode is and shows a picture. Make a note that it is enabled by default.